src/components/CheckboxInput/CheckboxInput.tsx

```diff
@@ -9,7 +9,55 @@ import './CheckboxInput.scss';
  * Used to allow users to make a range of selections (zero, one or many).
  */
 
-const CheckboxInput = ({
+const propTypes = {
+  /**
+   * The id attribute of the input
+   */
+  id: PropTypes.string.isRequired,
+  /**
+   * Additional classes to add
+   */
+  className: PropTypes.string.isRequired,
+  /**
+   * Mark the input field as invalid and display a validation message.
+   * Pass a string or node to render a validation message below the input
+   */
+  error: PropTypes.oneOfType([
+    PropTypes.bool,
+    PropTypes.string,
+    PropTypes.node,
+  ]).isRequired,
+  /**
+   * The checkbox input "checked" attribute
+   */
+  isChecked: PropTypes.bool.isRequired,
+  /**
+   * If the input should be disabled and not focusable
+   */
+  isDisabled: PropTypes.bool.isRequired,
+  /**
+   * Determines if input is required or not. (Label will have an asterisk if required)
+   */
+  isRequired: PropTypes.bool.isRequired,
+  /**
+   * Callback function when input is blurred.
+   */
+  onBlur: PropTypes.func.isRequired,
+  /**
+   * Callback function when input is changed
+   */
+  onChange: PropTypes.func.isRequired,
+  /**
+   * Callback function when input is focused
+   */
+  onFocus: PropTypes.func.isRequired,
+  /**
+   * Custom content to be displayed to right of checkbox. Can be any valid node/tree, anchors, etc.
+   */
+  label: PropTypes.node.isRequired,
```

Comment:
@juanfabrega so this is both used for the `aria-label (string | undefined)`, as well as passed into `FormLabel (node)`. we might have to restrict the label typing here to `string` if we want it to satisfy both

Reply:
@ericyliu this was updated on `develop` branch a few days ago to take both a string and a node. in the `FormLabel` component. Let me know if you don't see the same on your end.

Reply:
@ericyliu oh I see what you are saying now. Might as well follow the same proptype we have on `FormLabel` here as well. Can you chance it?

Reply:
@juanfabrega thats what im saying, the proptype on formlabel is `node`, but that doesnt work for aria-label